metadata.rb

```diff
@@ -1,7 +1,7 @@
 name             'pgpool'
-maintainer       'Jens Jorritsma'
-maintainer_email 'jens.jorritsma@gamil.com'
+maintainer 'Jens Jorritsma'
+maintainer_email 'jens.jorritsma@rackspace.com'
```

Comment:
Unless the maintainer request his/her email to be changed, lets not change this.